src/cloud-sql-instance.ts

```diff
@@ -43,6 +43,13 @@ interface CloudSQLInstanceOptions {
   sqlAdminFetcher: Fetcher;
 }
 
+interface RefreshableValues {
```

Comment:
should this be named something different for consistency? Is this the equivalent of [`refreshResult`](https://github.com/GoogleCloudPlatform/cloud-sql-go-connector/blob/main/internal/cloudsql/refresh.go#L289) or [`metadata`](https://github.com/GoogleCloudPlatform/cloud-sql-go-connector/blob/main/internal/cloudsql/refresh.go#L47) in Go?

Reply:
`RefreshResult` LGTM 👍